packages/devextreme-vue/src/core/component.ts

```diff
@@ -44,6 +44,7 @@ export interface IBaseComponent extends ComponentPublicInstance, IWidgetComponen
 }
 
 const includeAttrs = ['id', 'class', 'style'];
+const dxClasses: string[] = [];
```

Comment:
This is going to be one "static" array for all the Vue components on the page, which does not seem good